packages/snap-preact-components/src/components/Atoms/SearchHeader/readme.md

```diff
@@ -0,0 +1,75 @@
+# Search Header
+
+Renders a search header.
```

Comment:
Maybe a little more about the different titles it might display (landing page, results, no results, corrected queries, etc...).